### PR TITLE
feat: extend memory cache service

### PR DESCRIPTION
**Pull request by aochmann**
**What this PR does / why it's submitted / why we need it**:

Extended memory service, added unit tests (https://github.com/thecogworks/Cogworks.Essentials/issues/3) for implementation

---

**Checklist**:

- [x] My PR has a descriptive title (not a vague title like `Needed this`)
- [x] Title and all the commits follow [commitlint guidelines](https://github.com/conventional-changelog/commitlint#what-is-commitlint)
- [x] My PR targets the `develop` branch or appropriate `release` branch
- [ ] I've placed the link to the corresponding Trello card or issue which this PR fixes/address below ⤵️

---

### CODE REVIEW

**Review by ja0b — APPROVED** [review]

File: Source/Cogworks.Essentials/Constants/StringConstants.cs [hunk 1]
```diff
@@ -2,5 +2,6 @@
 {
     public static partial class StringConstants
     {
+        public const string CacheKeyList = "CacheKeyList";
```
Comment (aochmann) [inline Source/Cogworks.Essentials/Constants/StringConstants.cs:5]: To be removed. Not used anymore.

File: Source/Cogworks.Essentials/Constants/DateTimeConstants.cs [hunk 2]
```diff
@@ -16,6 +16,7 @@ public static class TimeInSecondsConstants
             public const int Minute = 60 * Second;
             public const int Hour = 60 * Minute;
             public const int Day = 24 * Hour;
+            public const int Year = Day * 360;
```
[the remark this answers is not in the document]
Comment (ja0b) [reply Source/Cogworks.Essentials/Constants/DateTimeConstants.cs:19]: 365

**Review by mzajkowski — APPROVED** [review]

**Comment by mzajkowski** [comment]
https://media0.giphy.com/media/9JyTQrfpJs8zZ9xLI3/giphy.gif?cid=ecf05e47m53j7iqlvvbbgmw44j336h264270i78g3yr4br6v&rid=giphy.gif&ct=g

**Review by lkoruba — APPROVED** [review]

**Pull request merged**